### DISCUSSION TITLE
Requesting Help

### DISCUSSION THREAD

**Discussion by IsraelGebresilasieKimo** [opengeos/mapwidget#5] [Q&A] [answered]
I was flowing an example on      https://mapwidget.gishub.org/examples/cesium/  
but I Don't Understand  'token'
![image](https://user-images.githubusercontent.com/106670155/226841856-4ecd2746-af15-451f-a1fb-921c4da32eae.png)
and
![image](https://user-images.githubusercontent.com/106670155/226841984-0867bddf-73c5-4968-83d6-98dee6ae5ab2.png)
  
and also can this code work for all parts of the globe?

any help?
hope to hear from you
with regards
Israel G.

**Comment by giswqs** [comment, marked as answer]
See the updated notebook for instructions: https://mapwidget.gishub.org/examples/cesium/
